beacon_node/beacon_chain/src/beacon_chain.rs

```diff
@@ -33,6 +38,12 @@ use types::*;
 //                          |-------must be this long------|
 pub const GRAFFITI: &str = "sigp/lighthouse-0.0.0-prerelease";
 
+/// If true, everytime a block is processed the pre-state, post-state and block are written to SSZ
+/// files in the temp directory.
+///
+/// Only useful for testing.
+const WRITE_BLOCK_PROCESSING_SSZ: bool = true;
```

Comment:
This would be nice behind a feature flag, `const WRITE_BLOCK_PROCESSING_SSZ: bool = cfg!(feature = "write_block_processing_ssz")`